yt_dlp/extractor/godresource.py

```diff
@@ -0,0 +1,85 @@
+from .common import InfoExtractor
+from ..utils import (
+    determine_ext,
+    merge_dicts,
+    unified_timestamp,
+    url_or_none
+)
+from ..utils.traversal import traverse_obj
+
+
+class GodResourceIE(InfoExtractor):
+    _VALID_URL = r'https?://new\.godresource\.com/video/(?P<id>\w+)'
+    _TESTS = [{
+        # hls stream
+        'url': 'https://new.godresource.com/video/A01mTKjyf6w',
+        'info_dict': {
+            'id': 'A01mTKjyf6w',
+            'ext': 'mp4',
+            'view_count': int,
+            'timestamp': 1710978666,
+            'channel_id': 5,
+            'thumbnail': 'https://cdn-02.godresource.com/e42968ac-9e8b-4231-ab86-f4f9d775841f/thumbnail.jpg',
+            'channel': 'Stedfast Baptist Church',
+            'upload_date': '20240320',
+            'title': 'GodResource',
```

Comment:
This is worse than the auto-generated title 'GodResource video #A01mTKjyf6w' and should not be used imo

Reply:
Resolved in d493ce72b63483e20d8001f8eabcc286cd4eb3b2